src/support/timefuncs.c

```diff
@@ -28,31 +28,32 @@
 extern "C" {
 #endif
 
-#if defined(_OS_WINDOWS_)
-static double floattime(void)
+int jl_gettimeofday(struct jl_timeval *jtv)
 {
-    struct timeb tstruct;
-
-    ftime(&tstruct);
-    return (double)tstruct.time + (double)tstruct.millitm/1.0e3;
-}
+#if defined(_OS_WINDOWS_)
+    struct __timeb64 tb;
+    errno_t code = _ftime64_s(&tb);
+    jtv->sec = tb.time;
+    jtv->usec = tb.millitm * 1000;
 #else
-static double tv2float(struct timeval *tv)
+    struct timeval now;
+    int code = gettimeofday(&now, NULL);
+    jtv->sec = now.tv_sec;
+    jtv->usec = now.tv_usec;
+#endif
+    return code;
+}
+
+static double jtv2float(struct jl_timeval *jtv)
 {
-    return (double)tv->tv_sec + (double)tv->tv_usec/1.0e6;
+    return (double)jtv->sec + (double)jtv->usec*1.0e-6;
 }
-#endif
 
 double clock_now(void)
 {
-#if defined(_OS_WINDOWS_)
-    return floattime();
-#else
-    struct timeval now;
-
-    gettimeofday(&now, NULL);
-    return tv2float(&now);
-#endif
+    struct jl_timeval now;
+    jl_gettimeofday(&now);
+    return jtv2float(&now);
```

Comment:
Is this the only place where we call `jtv2float`?  If so, you might as well just inline that function here.